src/components/shared/TheEntries.vue

```diff
@@ -4,22 +4,36 @@
     :data-test="props.entries ? 'entries' : ''"
     style="padding-bottom: 7rem"
   >
-    <h2 class="q-my-auto text-bold text-h5 q-pa-md">Entries</h2>
+    <h2 class="q-my-auto text-bold text-h5 q-pa-md">
+      Entries
+      <q-icon name="add" size="30px" @click="openEntryDialog()"><q-tooltip>Add entries</q-tooltip></q-icon>
```

Comment:
I would suggest adding "cursor: pointer" to the + button for better user experience